src/reports/Nonprofit_Edition_Reports/NPSP_Campaign_Household_Mailing_List_V2.report

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@judisohn The new report I created. Effectively identical to the old report that was in the Households package with a couple key changes:
* New report folder (needed because I can't install a new report from this package in a folder created by the Households package).
* Report field filter used instead Scope.

I kept the same report label because ideally the old report label will be changed by the Household package push, though I'm not 100% sure of that. I wasn't able to find this documented clearly.

Reply:
@force2b I just pushed this to a scratch org and for some reason, the report is not showing with V2 in the name. I think I'd prefer the folder be named something other than 'Nonprofit Edition' reports since we never call it 'Nonprofit Edition' anywhere. Maybe rename 'NPSP Health Check' to 'NPSP Settings Reports' and it will include health check and anything that comes out of 'system' type reports. @smarovich?

![image](https://user-images.githubusercontent.com/4614343/32275546-04b88aba-bee2-11e7-9227-8a553f07d22c.png)

As a result, had to guess which one: 

![image](https://user-images.githubusercontent.com/4614343/32275669-83abe970-bee2-11e7-849f-f67755780188.png)

Also, we'll need to change the text below to indicate the new report name.

Reply:
Yep. That makes sense based on what I posted just before this. We can't rename either during the deploy, so let's remove both the report and folder from this package and use "V2" in the report label for the new report in the Cumulus package. What do you think?

Reply:
please don't go renaming the NPSP Health Check folder.  I believe we have multiple messages and doc that refer to specific reports in that specific folder.

Reply:
This PR shouldn't touch that folder. I'm just removing the Campaign Household Mailing List report from the "Nonprofit Edition Reports" folder, which I think is actually deployed in a different package.

Reply:
Okay @davidhabib - I would just prefer we didn't call it 'Nonprofit Edition Reports' since this would be a good time to rename that folder. But if not, fine.

Reply:
@judisohn and @davidhabib We can't rename the folder that was installed with npe01, but the new report can either go into a new folder (that's what I did here) or we might be able to deploy into the already existing 'Nonprofit Edition Reports' folder that was installed with npe01. I was having difficulty getting IntelliJ to pull down the report metadata when it was in the npe01 folder, but that should work since the Households package (npo02) was deploying the report into the `npe01__Nonprofit_Edition_Reports` folder